docs/source/release_notes/release-notes.rst

```diff
@@ -146,12 +173,6 @@ develop
            (`Pull Request <https://github.com/palantir/atlasdb/pull/2003>`__)
 
 
-    *    - |devbreak| |improved|
-         - Converted all compile time uses of Guava's ``com.google.common.base.Optional`` class to the Java8 equivalent ``java.util.Optional``.
-           Should not affect deployments as there is no change to `json` or `yml` representations of AtlasDB configurations.
-           This is a relatively straightforward compile time break for products consuming AtlasDB libraries.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/2016>`__)
-
```

Comment:
Ah, no worries. This is what happens when we don't release for too long - the old version headings get hidden because they're too far away 😄